data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/SimpleCrac.java

```diff
@@ -252,7 +198,7 @@ public final State getState(String id) {
     @Override
     @JsonIgnore
     public State getPreventiveState() {
-        return states.values().stream().filter(state -> state.getContingency().isEmpty()).findAny().orElse(null);
+        return states.values().stream().filter(state -> state.getInstant().equals(Instant.PREVENTIVE)).findAny().orElse(null);
```

Comment:
maybe we could add a method isBaseCase() in State.
We are doing often that kind of check

Reply:
(we could add it later, in another PR)